module/VuFind/config/module.config.php

```diff
@@ -392,6 +392,7 @@
             'VuFind\Config\PluginManager' => 'VuFind\Config\PluginManagerFactory',
             'VuFind\Config\SearchSpecsReader' => 'VuFind\Config\YamlReaderFactory',
             'VuFind\Config\YamlReader' => 'VuFind\Config\YamlReaderFactory',
+            'VuFind\Connection\ExternalVuFind' => 'Laminas\ServiceManager\Factory\InvokableFactory',
```

Comment:
Those things would go in VuFind\Recommend\PluginManager, not here in module.config.php. In any case, yes, I would recommend adding them. While the default fallbacks will make things work, I try to avoid magic by providing explicit configuration where practical.

Reply:
Done.